CHANGELOG.md

```diff
@@ -14,7 +14,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   ([#204](https://github.com/Open-EO/openeo-python-client/issues/204))
 - Add `optional` argument to `Parameter` and fix re-encoding parameters with default value. (EP-3846)
 - Add support to test strict equality with `ComparableVersion`
-- Jupyter integration: add rich HTML rendering for more backend metadata (Job, Job Estimate, Logs, User-Defined Processes)
+- Jupyter integration: add rich HTML rendering for more backend metadata (Job, Job Estimate, Logs, Services, User-Defined Processes)
+- Added class `Service` for secondary web-services instead of returning a `dict` for them
```

Comment:
This should go under "Changed" I think, and be more explicit that the return type of `tiled_viewing_service`/`create_service` changed

Reply:
Okay, I've added more entries to the changelog, it should really be complete now.